types/find-replace/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for find-replace 4.0
+// Project: https://github.com/75lb/find-replace#readme
+// Definitions by: Renat Zamaletdinov <https://github.com/Zamaletdinov>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Replace or remove multiple items in an array
+ *
+ * `array` - The input array
+ *
+ * `findFn` - A predicate which, if returns true causes the current item to be operated on
+ *
+ * `...replaceWiths`:
+ *
+ * * If not specified, each found value will be removed.
+ * * If specified, each found value will be replaced with this value.
+ * * If the replaceWith value is a function, it will be invoked with the found value and its result used as the replace value.
+ * * If the replaceWith function returns an array, the found value will be replaced with each item in the array (not replaced with the array itself).
+ */
+declare function findReplace<T>(array: T[], findFn: (x: T) => boolean, ...replaceWiths: Array<T | ((x: T) => T)>): T[];
+declare function findReplace(array: any[], findFn: (x: any) => boolean, ...replaceWiths: any[]): any[];
+
```

Comment:
I think due to this line:
```md
Old browser (adds window.findReplace):
```
https://github.com/75lb/find-replace#readme
the package should be UMD like, allowing JS usage 
```suggestion

export as namespace findReplace;
```